examples/app/pivot/csv/electricity generation.csv

```diff
@@ -0,0 +1,4903 @@
+Modeled,IA,coal,2030,38753263.06
+Modeled,IA,gas,2010,268638.0547
+Modeled,IA,gas,2012,119519.4752
+Modeled,IA,gas,2014,87840.437
+Modeled,IA,gas,2016,480408.7463
+Modeled,IA,gas,2018,2677674.974
+Modeled,IA,gas,2020,679947.2889
+Modeled,IA,gas,2022,1903648.468
+Modeled,IA,gas,2024,2473237.154
+Modeled,IA,gas,2026,2630954.591
+Modeled,IA,gas,2028,2453546.373
+Modeled,IA,gas,2030,1712743.339
+Modeled,IA,nuclear,2010,4893209.856
+Modeled,IA,nuclear,2012,4893209.856
+Modeled,IA,nuclear,2014,4893209.856
+Modeled,IA,nuclear,2016,4893209.856
+Modeled,IA,nuclear,2018,4893209.856
+Modeled,IA,nuclear,2020,4893209.856
+Modeled,IA,nuclear,2022,4893209.856
+Modeled,IA,nuclear,2024,4893209.856
+Modeled,IA,nuclear,2026,4893209.856
+Modeled,IA,nuclear,2028,4893209.856
+Modeled,IA,nuclear,2030,4893209.856
+Modeled,IA,other,2010,51263.52
+Modeled,IA,other,2012,51263.52
+Modeled,IA,other,2014,73233.6
+Modeled,IA,other,2016,73233.6
+Modeled,IA,other,2018,73233.6
+Modeled,IA,other,2020,73233.6
+Modeled,IA,other,2022,73233.6
+Modeled,IA,other,2024,73233.6
+Modeled,IA,other,2026,73233.6
+Modeled,IA,other,2028,73233.6
+Modeled,IA,other,2030,73233.6
+Modeled,IA,solar,2010,258.6906861
+Modeled,IA,solar,2012,256.0591241
+Modeled,IA,solar,2014,15477.60138
+Modeled,IA,solar,2016,40794.86131
+Modeled,IA,solar,2018,89954.97109
+Modeled,IA,solar,2020,173915.5115
+Modeled,IA,solar,2022,308688.1594
+Modeled,IA,solar,2024,443125.9934
+Modeled,IA,solar,2026,543119.5775
+Modeled,IA,solar,2028,611518.0137
+Modeled,IA,solar,2030,656966.4094
+Modeled,IA,water,2010,575864.5078
+Modeled,IA,water,2012,575864.5078
+Modeled,IA,water,2014,575864.5078
+Modeled,IA,water,2016,575864.5078
+Modeled,IA,water,2018,575864.5078
+Modeled,IA,water,2020,1507186.022
+Modeled,IA,water,2022,3248402.312
+Modeled,IA,water,2024,3475074.082
+Modeled,IA,water,2026,3729338.15
+Modeled,IA,water,2028,3729338.15
+Modeled,IA,water,2030,3729338.15
+Modeled,IA,wind,2010,9675355.176
+Modeled,IA,wind,2012,15454447.04
+Modeled,IA,wind,2014,17487514.81
+Modeled,IA,wind,2016,17880926.91
+Modeled,IA,wind,2018,17875401.46
+Modeled,IA,wind,2020,17871297.16
+Modeled,IA,wind,2022,17865073.25
+Modeled,IA,wind,2024,17301788.03
+Modeled,IA,wind,2026,18824220.09
+Modeled,IA,wind,2028,31107253.42
+Modeled,IA,wind,2030,53013574.33
+Modeled,ID,bio,2010,95533.88645
+Modeled,ID,bio,2012,140833.3006
+Modeled,ID,bio,2014,140833.3006
+Modeled,ID,bio,2016,140833.3006
+Modeled,ID,bio,2018,140833.3006
+Modeled,ID,bio,2020,140833.3006
+Modeled,ID,bio,2022,140833.3006
+Modeled,ID,bio,2024,140833.3006
+Modeled,ID,bio,2026,140833.3006
+Modeled,ID,bio,2028,140833.3006
+Modeled,ID,bio,2030,140833.3006
+Modeled,ID,gas,2010,55487.08661
+Modeled,ID,gas,2012,3370555.952
+Modeled,ID,gas,2014,4482605.489
+Modeled,ID,gas,2016,2987877.963
+Modeled,ID,gas,2018,2478653.797
+Modeled,ID,gas,2020,2717424.024
+Modeled,ID,gas,2022,2504264.076
+Modeled,ID,gas,2024,2753007.082
+Modeled,ID,gas,2026,2782479.991
+Modeled,ID,gas,2028,2917963.431
+Modeled,ID,gas,2030,
```

Comment:
This may be a bit large to check into the repo directly. We should consider putting it in `bokeh.sampledata` and using it from there (and using the `mtcars` data from there as well)